share/completions/ansible.fish

```diff
@@ -0,0 +1,106 @@
+# fish 2.2.0 does not include native snippet support. Upgrade to fish >= 2.3.0
+# or append the following code to your ~/.config/fish/config.fish.
+
+# for file in ~/.config/fish/conf.d/*.fish
+#     source $file
+# end
+# ansible
+complete -c ansible -s a -l args -r -f -d "module arguments"
+complete -c ansible -l ask-vault-pass -f -d "ask for vault password"
+complete -c ansible -s B -l background -r -f -d "run asynchronously, failing after X seconds (default=N/A)"
+complete -c ansible -s C -l check -f -d "don't make any changes; instead, try to predict some of the changes that may occur"
+complete -c ansible -s D -l diff -f -d "when changing (small) files and templates, show the differences in those files; works great with --check"
+complete -c ansible -s e -l extra-vars -r -d "set additional variables as key=value or YAML/JSON"
+complete -c ansible -s f -l forks -a "(seq 0 (cat /proc/sys/kernel/pid_max))" -d "specify number of parallel processes to use (default=5)"
+complete -c ansible -s h -l help -d "shows a help message"
+complete -c ansible -s i -l inventory -r -d "specify inventory host path (default=/etc/ansible/hosts) or comma separated host list."
+complete -c ansible -s l -l limit -r -d "further limit selected hosts to an additional pattern"
+complete -c ansible -l limit-hosts -r -d "outputs a list of matching hosts; does not execute anything else"
+complete -c ansible -s m -l module-name -r -d "module name to execute (default=command)"
+complete -c ansible -s M -l module-path -r -d "specify path(s) to module library (default=None)"
+complete -c ansible -l new-vault-password-file -f -d "new vault password file for rekey"
+complete -c ansible -s o -l one-line -d "condense output"
+complete -c ansible -l output -f -d "output file name for encrypt or decrypt; use - for stdout"
+complete -c ansible -s P --l poll -r -d "set the poll interval if using -B (default=15)"
+complete -c ansible -l syntax-check -f -d "perform a syntax check on the playbook, but do not execute it"
+complete -c ansible -s t -l tree -f -a "(find . -maxdepth 1 -type d)" -d "log output to this directory"
```

Comment:
Use `__fish_complete_directories` for this and other options that may take a directory (but not a file).